src/dms/dmsBlockData.hpp

```diff
@@ -213,58 +190,30 @@ namespace engine
          _blockID = mbID ;
          _firstExtentID = DMS_INVALID_EXTENT ;
          _lastExtentID  = DMS_INVALID_EXTENT ;
-         _numIndexes    = 0 ;
          for ( i = 0 ; i < _max ; ++i )
          {
             _deleteList[i].reset() ;
          }
-         for ( i = 0 ; i < DMS_COLLECTION_MAX_INDEX ; ++i )
-         {
-            _indexExtent[i] = DMS_INVALID_EXTENT ;
-         }
+
          _logicalID = clLID ;
-         _indexHWCount = 0 ;
          _attributes   = attr ;
          _loadFirstExtentID = DMS_INVALID_EXTENT ;
          _loadLastExtentID  = DMS_INVALID_EXTENT ;
          _mbExExtentID      = DMS_INVALID_EXTENT ;
 
          _totalRecords           = 0 ;
          _totalDataPages         = 0 ;
-         _totalIndexPages        = 0 ;
          _totalDataFreeSpace     = 0 ;
-         _totalIndexFreeSpace    = 0 ;
-         _totalLobPages          = 0 ;
-         _totalLobs              = 0 ;
-         _compressorType         = UTIL_COMPRESSOR_INVALID ;
          _dictVersion            = 0 ;
```

Comment:
we don't support compression yet, guess no need dictionary.  but maybe need it in the future.

Reply:
leave the dictionary for now